packages/did-ipfs-store/src/didStore.ts

```diff
@@ -27,9 +33,22 @@ export class DidStore implements IDidStore {
     private headers: Record<string, string> = {}
   ) {}
 
-  async save(claim: string): Promise<string> {
+  /**
+   * @param claim stringified claim. Supported types of claim content are `string` and `object`
+   * @param pinTimeout Defines how to long to wait for pinning completion before throwing error, seconds
+   */
+  async save(claim: string, pinTimeout = 5): Promise<string> {
     const blob = new Blob([claim]);
     const { cid } = await this.add(blob);
+
+    const { default: waitFor } = await (eval('import(`p-wait-for`)') as Promise<
```

Comment:
@JGiter will use of dynamic import cause issues with the jest (https://energyweb.atlassian.net/browse/SWTCH-1508?focusedCommentId=23518) ?

Reply:
Ah, right. Will try `wait-on` instead

Reply:
Ah, yeah it is a problem. I will have to implement it then